src/CONST.js

```diff
@@ -170,11 +170,26 @@ const CONST = {
             shortcutKey: 'Enter',
             modifiers: [],
         },
+        CTRL_ENTER: {
+            descriptionKey: null,
+            shortcutKey: 'Enter',
+            modifiers: ['CTRL'],
+        },
         COPY: {
             descriptionKey: 'copy',
             shortcutKey: 'C',
             modifiers: ['CTRL'],
         },
+        ARROW_UP: {
+            descriptionKey: null,
+            shortcutKey: 'ArrowUp',
+            modifiers: [],
+        },
+        ARROW_DOWN: {
+            descriptionKey: null,
```

Comment:
If you pass a `descriptionKey`, that will cause the keyboard shortcut to show up in here:

<img width="615" alt="image" src="https://user-images.githubusercontent.com/47436092/163493319-215c4ede-1462-4856-9f6b-4b76d9ae7e17.png">

If you add a new keyboard shortcut, then you need a `descriptionKey` if and only if you want to document what it does in that modal. Since these are more generic it would be hard to consistently describe what they do without causing more confusion.

Reply:
Oh hmm that is kind of weird and makes me think a "keyboard shortcut" is different thing from a "keyboard event/action/whatever". Like all keyboard shortcuts are keyboard events but not all keyboard events are "shortcuts"? Anyway NAB.